### PR TITLE
Node specific configurations for StorageCluster spec

### PR DESCRIPTION
**Pull request by piyush-nimbalkar**
Signed-off-by: Piyush Nimbalkar <piyush@portworx.com>

### CODE REVIEW

File: content/reference/CRD/storage-cluster.md [hunk 1]
```diff
@@ -259,3 +290,18 @@ This section provides details on how to deploy and manage Autopilot.
...
+
+### Node specific configuration
+
+This section provides details on how to override certain cluster level configuration for individual or group of nodes.
+
+| Field | Description | Type | Default |
+| --- | --- | --- | --- |
+| spec.<br>nodes[] | A list of node specific configurations. | []object | None` |
+| spec.<br>nodes[].<br>selector | Selector for the node(s) to which the configuration in this section will be applied. | object | None |
+| spec.<br>nodes[].<br>selector.<br>.nodeName | Name of the node to which this configuration is applied. Node name takes precendence over `selector.labelSelector`. | string | None |
```
Comment (nathan-px) [inline content/reference/CRD/storage-cluster.md:302]: "is" or "will be"? We should keep the tense consistent unless there's a reason for it.

File: content/reference/CRD/storage-cluster.md [hunk 2]
```diff
@@ -259,3 +290,18 @@ This section provides details on how to deploy and manage Autopilot.
...
 | spec.<br>autopilot.<br>providers.<br>type | Type of the data provider. For instance, `prometheus` | `string` | None |
 | spec.<br>autopilot.<br>providers.<br>params | Map of key-value params for the provider. | `map[string]string` | None |
+
+### Node specific configuration
+
+This section provides details on how to override certain cluster level configuration for individual or group of nodes.
+
+| Field | Description | Type | Default |
+| --- | --- | --- | --- |
+| spec.<br>nodes[] | A list of node specific configurations. | []object | None` |
```
Comment (nathan-px) [inline content/reference/CRD/storage-cluster.md:300]: All of the existing "Type" column items are shown as `code`. I recommend making this consistent with those. 

```suggestion
| spec.<br>nodes[] | A list of node specific configurations. | `[]object` | None |
```

**Comment by piyush-nimbalkar** [comment]
@nathan-px Can you re-trigger the travis build on this? I don't seem to have access to do it.

**Comment by andreipope** [comment]
> @nathan-px Can you re-trigger the travis build on this? I don't seem to have access to do it.

@piyush-nimbalkar Done.

**Pull request merged**